src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs

```diff
@@ -225,20 +222,20 @@ private async Task RecomputeTagsAsync(bool highPriority, CancellationToken cance
                     var oldState = this.State;
 
                     var textChangeRange = this.AccumulatedTextChanges;
-                    this.AccumulatedTextChanges = null;
+                    var subjectBufferVersion = _subjectBuffer.CurrentSnapshot.Version.VersionNumber;
 
-                    // Technically not necessary since we ConfigureAwait(false) right above this.  But we want to ensure
-                    // we're always moving to the threadpool here in case the above code ever changes.
                     await TaskScheduler.Default;
 
-                    cancellationToken.ThrowIfCancellationRequested();
+                    if (cancellationToken.IsCancellationRequested)
+                        return;
```

Comment:
📝 I retained these changes as they represent a trivial consistency update to this method, and do not overlap with the planned compiler changes.